colorcoder/utils/ColorCoderUtils.java

```diff
@@ -0,0 +1,32 @@
+package colorcoder.utils;
+
+import colorcoder.constants.ColorConstants;
+import colorcoder.enums.MajorColor;
+import colorcoder.enums.MinorColor;
+import colorcoder.models.ColorPair;
+
+public class ColorCoderUtils {
```

Comment:
The name 'Utils' will attract much functionality over time. Try a more specific name to highlight the purpose of the functionality.